responsive-web-design-piano/index16.html

```diff
@@ -89,52 +24,48 @@
             <div class="key black--key"></div>
             <div class="key black--key"></div>
             <div class="key black--key"></div>
-
-            <div class="key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
-            <div class="key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
-            <div class="key black--key"></div>
         </div>
     </div>
- 
+
 </body>
 </html>
-<!-- 
-    Time to make the piano responsive by adding media queries. Media
-    queries uses the `@media` rule to include blocks of CSS properties
-    only if set screen condition is true. Start by adding below the pseudo
-    element CSS properties the following: `@media only screen and (max-width: 768px) {}`
-    This `@media` rule will target devices smaller then 768px. Adjust the `#piano`'s
-    size to fit in smaller screen devices by changing its `width` to `355px`. Also,
-    adjust the `.keys` `width` to `318px`. Finally adjust freeCodeCamp's `.logo`'s 
-    `width` to `150px`.
 
-    The code should look like so:
-    
-    ```
-    ...
-        /* media queries */
-
-        /* devices smaller then 768px */
-        @media only screen and (max-width: 768px) {
-
-            #piano {
-                width: 355px;
-            }
-
-            .keys {
-                width: 318px;
-            }
+<!-- 
+    We need a third set of 7 `key` `<div>` elements to complete
+    a total of 21 `key` `<div>` elements. Copy one of the set of seven
+    `key` `<div>` elements and paste another set below.
 
-            .logo {
-                width: 150px;
-            }
+    Here is an example:
```

Comment:
I'm thinking this example can be removed. It's basically a repeat of what the student did in the last step.